packages/webapp/pages/posts/[id].tsx

```diff
@@ -87,12 +86,6 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
     options: { initialData, retry: false },
   });
 
-  useEffect(() => {
-    if (!post?.private || !post?.permalink) return;
-
-    globalThis?.location.replace(post.permalink);
-  }, [post]);
-
```

Comment:
If we point to the permalink when clicking the post content summary box, not sure why we would not redirect from here.

https://github.com/dailydotdev/apps/blob/f148977b74d46f972f7e0abd43f3102db72e8aff/packages/shared/src/components/post/SquadPostContent.tsx#L148-L153

Reply:
Oh now I understand the notion of this snippet, but if this is the case than the original condition is completely not aligned with external link case. Anyway, I suggest for simplicity to remove the redirect. There's no way for a user to land on an external link post page unless they manually build the url

Reply:
I agree, landing here is highly unlikely. The condition for the "safety-net" just came out wrong on the implementation. Thanks for the fix!

Reply:
The concern was that links could get shared which is now possible and it should rather show 404 but decisions where made to redirect on slack before.